app/models/decidim/kids/minor_data.rb

```diff
@@ -13,6 +13,7 @@ class MinorData < ApplicationRecord
       encrypt_attribute :name, type: :string
       encrypt_attribute :email, type: :string
       encrypt_attribute :birthday, type: :string
+      encrypt_attribute :password, type: :string
```

Comment:
```suggestion
```

Let's not store the password here.
Also: to do this it would require a migration